source/material_model/steinberger.cc

```diff
@@ -187,13 +189,22 @@ namespace aspect
         delta_temperature = temperature-adiabatic_temperature;
 
       // For an explanation on this formula see the Steinberger & Calderwood 2006 paper
+      // We here compute the lateral variation of viscosity due to temperature (vis_lateral) as
+      // V_lT = exp [-(H/nR)*dT/(T_adiabatic*(T_adiabatic + dT))] as in Eq. 6 of the paper.
+      // We get H/nR from the lateral_viscosity_lookup->lateral_viscosity function.
       const double vis_lateral_exp = -1.0*lateral_viscosity_lookup->lateral_viscosity(depth)*delta_temperature/(temperature*adiabatic_temperature);
```

Comment:
Please rename `vis_lateral_exp` to `log_temperature_prefactor`

Reply:
A follow-up question on these lines.

The equation in the comments is written as `V_lT = exp [-(H/nR)*dT/(T_adiabatic*(T_adiabatic + dT))]`, while the actual equation on line 195 has `delta_temperature/(temperature*adiabatic_temperature)`.

Is the equation in the comments referring to the same equation as the one on line 195? If yes, can you confirm which one is correct or expand the documentation a bit further to explain the differences between the two equations?

Reply:
@naliboff I suggested to write the equation like this, this is how it is written in the Steinberger and Calderwood paper. 
`temperature` is the same as `T_adiabatic + dT`